clang/test/Sema/uninit-variables.c

```diff
@@ -551,3 +551,14 @@ struct full_of_empty empty_test_2(void) {
   struct full_of_empty e;
   return e; // no-warning
 }
+
+struct with_explicit_field {
+  int x;
+  int y [[clang::requires_explicit_initialization]]; // expected-note {{declared}}
```

Comment:
Noticed you didn't update the diagnostics check lines like I requested.  Please limit partial matches to much less than this.  Also, put the notes right next to the diagnostic, and use the `marker` feature to keep the notes together with the diagnostic (see here for how to do so https://clang.llvm.org/docs/InternalsManual.html#specifying-diagnostics).